cvs_parse_base_Implementation

DDemoNZ opened the pull request:
add&config swagger
add Apache commons csv parser
add ReviewRepository ReviewService
add implementation csv parsing ReviewReadParseInsertController

DDemoNZ committed:
add&config swagger
add Apache commons csv parser
add ReviewRepository ReviewService
add implementation csv parsing ReviewReadParseInsertController

DDemoNZ committed:
add&config swagger
add Apache commons csv parser
add ReviewRepository ReviewService
add implementation csv parsing ReviewReadParseInsertController

Tarabarnik commented on src/main/java/mate/dev/boot/csvparse/util/CsvUtil.java:
```diff
@@ -0,0 +1,44 @@
+            "HelpfulnessNumerator",
+            "HelpfulnessDenominator", "Score", "Time", "Summary", "Text",};
+
+    public static List<Review> read() throws IOException {
```
давай позбудемося статики? зроби цей клас компонентом та автовайри його в класах, де використовуються його методи

Tarabarnik commented on src/main/java/mate/dev/boot/csvparse/util/CsvUtil.java:
```diff
@@ -0,0 +1,44 @@
+            review.setText(record.get("Text"));
+            reviews.add(review);
+        }
+        return reviews;
```
чи можна це зробити через стріми?

DDemoNZ replied:
А зачем если это только для этого метода 
Выносит метод для стрима есть смысл?

DDemoNZ committed:
add SecurityConfig and DB auth/reg
add GetMostActiveUsers method and pagination
fix some stylistic things and checkstyle

DDemoNZ committed:
add getMostPopularWords method and getMostCommentedProducts method
add ProductResponseDto and MostPopularWordResponseDto

Tarabarnik commented on src/main/java/mate/dev/boot/csvparse/security/CustomUserDetailsService.java:
```diff
@@ -0,0 +1,38 @@
+                    .map(Role::getRoleName)
+                    .toArray(String[]::new));
+            return userBuilder.build();
+        } else {
```
навіщо тут else?
